core/src/main/java/ai/timefold/solver/core/impl/score/director/InnerScoreDirectorFactory.java

```diff
@@ -26,34 +23,10 @@ public interface InnerScoreDirectorFactory<Solution_, Score_ extends Score<Score
      */
     ScoreDefinition<Score_> getScoreDefinition();
 
-    @Override
-    default InnerScoreDirector<Solution_, Score_> buildScoreDirector(boolean lookUpEnabled,
-            ConstraintMatchPolicy constraintMatchPolicy) {
-        return buildScoreDirector(lookUpEnabled, constraintMatchPolicy, true);
-    }
-
-    @Override
-    InnerScoreDirector<Solution_, Score_> buildScoreDirector(boolean lookUpEnabled, ConstraintMatchPolicy constraintMatchPolicy,
-            boolean expectShadowVariablesInCorrectState);
+    AbstractScoreDirector.AbstractScoreDirectorBuilder<Solution_, Score_, ?, ?> createScoreDirectorBuilder();
```

Comment:
Should this method be part of `ScoreDirectorFactory`?

Reply:
Yes. How else would you create the builder without knowing that it exists?
It's a score director factory. It makes sense that it would provide means of production a score director.